src/AppInstallerCLITests/InstallFlow.cpp

```diff
@@ -536,6 +536,30 @@ TEST_CASE("MSStoreInstallFlowWithTestManifest", "[InstallFlow][workflow]")
     REQUIRE(installResultStr.find("9WZDNCRFJ364") != std::string::npos);
 }
 
+TEST_CASE("MSStoreInstallFlowWithTestManifestBypassStoreClientBlockedByPolicy", "[InstallFlow][workflow]")
+{
+    TestCommon::TempFile installResultPath("TestMSStoreInstalled.txt");
+
+    std::ostringstream installOutput;
+    TestContext context{ installOutput, std::cin };
+    auto previousThreadGlobals = context.SetForCurrentThread();
+    OverrideForMSStore(context, false);
+    context.Args.AddArg(Execution::Args::Type::Manifest, TestDataFile("InstallFlowTest_MSStore.yaml").GetPath().u8string());
+    context.SetFlags(Execution::ContextFlag::BypassIsStoreClientBlockedPolicyCheck);

```

Comment:
I think if there's no previous setup to disable the policy, the related code path is not tested. we should do the proper policy disable setup(and revert in test end) and do a positive, negative install attempt with and without the flag.